queries/roc/textobjects.scm

```diff
@@ -0,0 +1,35 @@
+(anon_fun_expr
+  (expr_body) @function.inner
+) @function.outer
+
+(argument_patterns
+  ((_) @parameter.inner . ","? @parameter.outer) @parameter.outer
```

Comment:
This query makes it so that `@parameter.outer` only matches the comma sometimes (maybe every time?). Can you change this to only have the second outer query which captures everything? Or if that doesn't work you may need `#make-range!`

Reply:
I don't understand what the problem is here. A Roc function looks like this:

```
\x -> x
```

or with multiple params:

```
\x, y -> x
```

Reply:
Yes; if you try it yourself, you'll find that operating on `@parameter.outer` selects only the comma itself, when it should be the entirety of, say, `x,` (with or without trailing whitespace). This is why you may need `#make-range!`. You can see some examples of other textobject files doing this if you need, this type of query is pretty common across different languages.

Reply:
thanks for explaining. this is my first foreay into tree-sitter